### PR TITLE
LIME experiments

### PR DESCRIPTION
**Pull request by FHoltorf**
Currently still experimenting with using LIME 

ToDo

- [x] Use logistic regression as opposed to linear regression on logits
- [x] Use new distance data
- [x] Extract label specific importance rankings
- [x] implement visualization routines

### CODE REVIEW

**Comment by FHoltorf** [comment]
Logistic regression did not work as intended. sklearn sadly does not appear to accommodate general cross entropy loss with uncertain class labels.

**Comment by FHoltorf** [comment]
I ran all LIME experiments for both data splits but only on the *test* data. The latter choice was made for various reasons: 1) It takes a while to run and this was faster for now. 2) I'd argue that one may suspect that using the test data could potentially avoid spuriously assigning too much importance to features due to overfitting. 3) We have plenty of test data.
